src/navigators/BottomNavigator.js

```diff
@@ -32,7 +32,11 @@ const BottomNavigator = () => {
           .string(),
       }}
       theme={{ colors: theme }}
-      activeColor={theme.primary}
+      activeColor={
```

Comment:
According to Material 3 guidelines, `activeColor` should be On secondary container.